src/LodeRunner.API/src/Controllers/ClientsController.cs

```diff
@@ -70,30 +70,29 @@ public IActionResult GetClients([FromServices] ILRAPICache appCache, [FromServic
         [HttpGet("{clientStatusId}")]
         [SwaggerResponse((int)HttpStatusCode.OK, "Single `Client` document by clientStatusId.", typeof(Client), "application/json")]
         [SwaggerResponse((int)HttpStatusCode.BadRequest, SystemConstants.InvalidClientStatusId, typeof(Middleware.Validation.ValidationError), "application/json")]
+        [SwaggerResponse((int)HttpStatusCode.NotFound, "`Data not found.`", null, "application/json")]
```

Comment:
Should be text/plain since it is not a JSON response.

Reply:
you are right,  but that should be for GetClients (line 53) since it returns NoContent , GetClientById returns NotFound and we return a JsonResult.